library/platform_util.c

```diff
@@ -26,18 +26,44 @@
 #define _POSIX_C_SOURCE 200112L
 #endif
 
+#if !defined(_GNU_SOURCE)
+/* Clang requires this to get support for explicit_bzero */
+#define _GNU_SOURCE
+#endif
+
 #include "common.h"
 
 #include "mbedtls/platform_util.h"
 #include "mbedtls/platform.h"
 #include "mbedtls/threading.h"
 
 #include <stddef.h>
+
+#ifndef __STDC_WANT_LIB_EXT1__
+#define __STDC_WANT_LIB_EXT1__ 1 /* Ask for the C11 gmtime_s() and memset_s() if available */
+#endif
 #include <string.h>
 
+#if defined(_WIN32)
+#include <windows.h>
+#endif
+
+// Detect platforms known to support explicit_bzero()
+#if defined(__GLIBC__) && (__GLIBC__ >= 2) && (__GLIBC_MINOR__ >= 25) \
+    && !defined(__ARM_EABI__)
```

Comment:
It does. I needed it to address a CI failure but I can't remember exactly what variant was failing. From local testing just now, I think this may have been resolved by the later commit which defines `_GNU_SOURCE`, so I think the EABI condition can be removed.

I will remove `&& !defined(__ARM_EABI__)` and push it through the CI again.

Reply:
Indeed requiring `_GNU_SOURCE` seems much more likely to me than an exception for one specific ABI on one specific processor.